Combat-Critters_2.0.csproj

```diff
@@ -56,12 +56,16 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="CombatCrittersSharp" Version="0.2.4" />
 		<PackageReference Include="CommunityToolkit.Maui" Version="9.1.0" />
 		<PackageReference Include="CommunityToolkit.Mvvm" Version="8.3.2" />
 		<PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiVersion)" />
 		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
+
+		<!-- Reference to the local DLL -->
+        <Reference Include="CombatCrittersSharp">
+            <HintPath>libs/CombatCrittersSharp.dll</HintPath>
+        </Reference>

```

Comment:
this can't be committed. the wrapper should still be referenced before being pulled into main